b2/console_tool.py

```diff
@@ -278,6 +278,7 @@ def run(self, args):
                 )
                 self.api.account_info.clear()
                 return 1
+            self.api.authorize_account(realm, args.applicationKeyId, args.applicationKey)
```

Comment:
This is a bug, not sure why it got added. 
Will remove it soon.

Reply:
This is fixed in https://github.com/Backblaze/B2_Command_Line_Tool/pull/570/commits/4e785fd1b0d793645668c62e9a3be835381c9311